From 96585612d17d31fa9dd0f5c260fc33f72be9a75f Mon Sep 17 00:00:00 2001
From: Jeremy Huddleston <jeremy@redxiii.local>
Date: Mon, 02 Feb 2009 05:09:44 +0000
Subject: XORG_CWARNFLAGS: += is not valid for appending in bourn scripts

Fixes:

./configure: line 12921: CWARNFLAGS+= -Wold-style-definition -Wdeclaration-after-statement: command not found
---
diff --git a/xorg-macros.m4.in b/xorg-macros.m4.in
index cbdcfa6..2a5e59c 100644
--- xorg-macros.m4.in
+++ xorg-macros.m4.in
@@ -447,7 +447,7 @@ if  test "x$GCC" = xyes ; then
 -Wbad-function-cast"
     case `gcc -dumpversion` in
     3.4.* | 4.*)
-	CWARNFLAGS+=" -Wold-style-definition -Wdeclaration-after-statement"
+	CWARNFLAGS="$CWARNFLAGS -Wold-style-definition -Wdeclaration-after-statement"
 	;;
     esac
 else
--
cgit v0.8.2
